package.json

```diff
@@ -17,7 +17,7 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "test:e2e": "jest --maxConcurrency=1 --config ./test/jest-e2e.json",
```

Comment:
I'm just curious, how does this impact our test suite?

Reply:
Does it solve the issue that is preventing the tests to exit gracefully?

Reply:
> I'm just curious, how does this impact our test suite?

I had some flaky test results when running the tests concurrently. Due to our db cleanup via the `afterAll` hook, basically a race condition. E.g. the e2e tests of the `Scraper Module` would finish, while the `Deposit Module` is still running and delete the deposit fixtures...

> Does it solve the issue that is preventing the tests to exit gracefully?

Unfortunately not... Not quite sure what is preventing a graceful exit tbh. I can dig down a bit as part of this PR or investigate separately?

Reply:
Interesting... if I remove the nested `describe` block, then it actually gracefully shuts down 🤔